.gitignore

```diff
@@ -0,0 +1,3 @@
+build/
+node_modules/
+package-lock.json
```

Comment:
I generally exclude lockfiles for libraries (because they're expected to work with *all* declared versions of their dependencies) and include them for applications (because it's valuable to ensure they run in production with the same dependencies as development).